README.md

```diff
@@ -1,23 +1,57 @@
-# Next.js + Jest
-
-This example shows how to configure Jest to work with Next.js.
-
-This includes Next.js' built-in support for Global CSS, CSS Modules, and TypeScript!
-
-## How to Use
-
-Quickly get started using [Create Next App](https://github.com/vercel/next.js/tree/canary/packages/create-next-app#readme)!
-
-In your terminal, run the following command:
-
-```bash
-npx create-next-app --example with-jest with-jest-app
-# or
-yarn create next-app --example with-jest with-jest-app
-```
-
-## Run Jest Tests
-
-```bash
-npm test
-```
+# Sparkbox Apprentice Page
+
+## Contributing 
+
+##### Git:
+
+- When you want to make a contribution please follow the [Sparkbox git process](https://github.com/sparkbox/standard/tree/main/code-style/git) for naming branches, writing commit messages, and rebasing before submitting PRs.
+
+##### Updating Apprentices Data:
+
+  ###### Images: 
+
+  - All image files should be added to ```/public/apprentices``` with the file name being ```[ApprenticeName].png```
+
+  ###### Apprentice Data:
+
+  - To add/change the current apprentices or previous apprentices data navigate to ```/data``` folder and find the ```apprentices.json``` file
+
+  - Add to and/or adjust the data, see below how the data for each group is structured:
+    - *Current Apprentices*:
+    ```js
+    "currentApprentices": {
```

Comment:
This is great, but technically will be inaccurate until we address https://sparkbox.atlassian.net/browse/FSA21V2-155

Let's get the rest of the updates in place, but hold off on merging until we address that.